modules/ROOT/pages/_partials/sync/js-client-offline-and-conflict.adoc

```diff
@@ -137,23 +137,36 @@ To resolve conflicts on the client side, a `conflictResolutionStrategy` needs to
 
 If none is provided, by default, the `clientVersionWins` strategy is used. This means the SDK will automatically override the server data with the current client data. 
 
-It is very easy to provide a custom conflict resolution strategy: it is a function that will take in the current server data, and client data as parameters, and the return value will be the resolved data. 
+It is very easy to provide a custom conflict resolution strategy. It takes two parameters, `strategies` and `default`.
```

Comment:
```suggestion
To provide a custom conflict resolution strategy, you only need to specify two parameters:
```

Reply:
I won't take this suggestion, but only because you've highlighted a different issue with what I wrote. Update on the way.

Reply:
yeah, try to avoid "it's very easy to do anything" - we've got marketing for that :wink: